web/packages/teleport/src/services/api/api.js

```diff
@@ -64,7 +65,18 @@ const api = {
     });
   },
 
-  fetchJson(url, params) {
+  async fetchJson(url, params, withMFA) {
+    if (withMFA) {
+      // Get an MFA response and add it to the request headers.
+      const webauthn = auth.getWebauthnResponse();
```

Comment:
This shouldn't work 🤔
We have to await the promise to get a valid response:
```suggestion
      const webauthn = await auth.getWebauthnResponse();
```